### DISCUSSION TITLE
long url for mauritania tv

### DISCUSSION THREAD

**Discussion by ARAB-IPTV** [iptv-org/community#133] [Show and tell]
https://www.youtube.com/watch?v=tinWeR_O8zE&ab_channel=%D9%82%D9%86%D8%A7%D8%A9%D8%A7%D9%84%D9%85%D9%88%D8%B1%D9%8A%D8%AA%D8%A7%D9%86%D9%8A%D8%A9T%C3%A9l%C3%A9visiondeMauritanie

**Comment by BellezaEmporium** [comment]
http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.youtube.com/channel/UCK3909NQ9meg2KDYKF51Kmg/live
